utils.py

```diff
@@ -78,11 +78,17 @@ def parse_qtop_cmdline_args():
                       help="Do not print results to stdout")
     parser.add_option("-r", "--removeemptycorelines", dest="REM_EMPTY_CORELINES", action="store_true", default=False,
                       help="If a whole row consists of empty core lines, remove the row")
-    parser.add_option("-R", "--replay", action="append", type="string", dest="REPLAY", default=[],
-                      help="instant replay from a specific moment in time for the cluster, and for a specific duration. "
-                           "The value inputted should be of the form <yyyymmddThhmmss>, e.g. -R 20161117T102300."
-                           "A second value is for now mandatory and denotes the desired length of the playback in minutes,"
-                           "e.g. -R 20161117T102300 -R2")
+    parser.add_option("-R", "--replay", action="callback", dest="REPLAY", callback=_watch_callback,
+                      help="instant replay from a specific moment in time for the "
+                           "cluster, and for a specified duration. The value "
+                           "inputted should be in either of the following formats: "
+                           "yyyymmddTHHMMSS, e.g. 20161118T182300, (explicit form) "
```

Comment:
so... this is indeed a very good PR.

I propose a filtering of some chars to make this expression more generic, fi: `''.join([x for x in a if x not in ':/-'])`